scripts/bulk-access-control.md

```diff
@@ -0,0 +1,93 @@
+# Bulk Access Control Management script
+[Back to the list of all defined scripts](./index.md)
+
+<!-- TOC -->
+* [Import script](#bulk-access-control-management-script)
+  * [What it does?](#what-it-does)
+  * [Who can use it?](#who-can-use-it)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion](#conclusion)
+* [What is an Access Condition?](#what-is-an-access-condition)
+<!-- TOC -->
+## What it does?
+
+This script performs bulk changes to the access conditions of the items and bitstreams. 
+The Bulk Access Control Management focus on Access Control rather than resource policies. The goal will be to add / replace access condition for a set of objects from a user friendly interface that will abstract from the complexity/internal rules of DSpace’s resource policies and bundles as much as possible.
+It takes a json file as input to identify the items and bitstreams target of the changes and the access conditions to append or set.
+
+## Who can use it?
+
+This script can be used by any user that has administrative priviled over at least one community, collection or item limited to the resources that they admin.
+
+## When we use it?
+
+If you need to align the access conditions of all the items and/or bitstreams in a community or collection, or other selection to new rules defined by the Institution such as making open access all the thesis or restrict access to all the material related to a specific project, etc.
+
+
+## How it Works?
+
+The Bulk Access Control Management script reads a json file containing the details about which resources (items and bitstreams) need to be processed and how to manipulate the access conditions.
+The script then processes the items and bitstreams identified according to the defined mode and specified accessConditions.
+
+## Parameters
+
+The script takes the following parameters:
+
+| Parameter                        | Description                                                                                                                                                                            |
+|----------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-f`, <br>`--file`                | This parameter specifies the json file containing the processing instruction.                                                                                                              |
+| `-h`, <br>`--help`               | This parameter displays help information for the script.                                                                                                                               |
```

Comment:
I would highly recommend we add a `-v` (`--validate-only`) option to this script which can be used to validate the JSON file & perhaps return a response of what changes *would have been made*.  This would be similar to how `metadata-import` has the same param to validate

Reply:
It seems fine to leave this `--validate-only` option for later since an invalid JSON file will throw an error.